scripts/lint.sh

```diff
@@ -76,8 +76,14 @@ fi
 CHANGED_FILES="$(git diff --name-only | grep '\.py$' | tr '\n' ' ')"
 if [ "$CHANGED_FILES" != "" ]
 then
+    RED="\033[0;31m"
+    echo "-------------------------------"
+    echo "${RED} [format] These files are not well-formatted:"
     git diff --name-only
-    echo "There are uncommitted changes on disk likely caused by the linters."
+    echo "-------------------------------"
+    echo "${RED} [format] Suggested format by lint:"
+    git diff
+    echo "${RED} If these changes will be synced to Meta's codebase, you must fix them because code format is enforced in Meta."
```

Comment:
aren't all changes synced? We generally prefer to avoid Meta specific comments/changes so would prefer to just remove this line

Reply:
that makes sense. Let me remove it.